app/models/apple/config.rb

```diff
@@ -27,6 +31,41 @@ class Config < ApplicationRecord
                                          message: "can only have one credential per public and private feed"}
     validates :public_feed, exclusion: {in: ->(apple_credential) { [apple_credential.private_feed] }}
 
+    def self.build_private_feed(podcast)
```

Comment:
Any case for erroring out if one already exists?  Or maybe... updating its attributes if it already exists?  Kind of a `build_or_update_private_feed`?

For the moment at least, we probably want just 1 of these per podcast.

Reply:
For sure!  Also, I added some loud warnings so that we can be aware, at least until https://github.com/PRX/feeder.prx.org/issues/793 lands, which should stem the danger in having multiple apple_config rows per podcast.